test/inductor/test_torchinductor_opinfo.py

```diff
@@ -194,6 +194,7 @@ def process(device_type):
     "bernoulli": {f32, f64},
     "bincount": {i32, i64},
     "bucketize": {b8, f16, f32, f64, i32, i64},
+    "cauchy": {f16, f64},
```

Comment:
Why does this add expected failures?

Reply:
So what's the story with inductor opinfo testing now? It can't be passing because results for random sampling would be different between eager and inductor. Is the test not run?

Reply:
> It can't be passing because results for random sampling would be different between eager and inductor. Is the test not run?

Yes, the tests will be skipped because eager --vs-- inductor randomness are not comparable. (The tests are not skipped due to graph break of inplace ops; graph break has been resolved.)
But I don't know why some inductor randomness tests (eg: uniform cpu float32, float64) have passed. https://github.com/pytorch/pytorch/pull/93243/files#r1094149843

I'm not sure how generator is partially supported in inductor.